### DISCUSSION TITLE
Enable log rotation

### DISCUSSION THREAD

**Discussion by KapilBishnoi** [prometheus/prometheus#11815] [Q&A]
I need to enable log rotation(https://prometheus.io/docs/guides/query-log/#rotating-the-query-log) for Prometheus but the utility logrotate is not present in the docker container based on ([quay.io/prometheus/prometheus:latest](http://quay.io/prometheus/prometheus:latest)). What is the recommended way to enable log rotation in the containers?

**Comment by roidelapluie** [comment]
In general, log rotation for containers is typically handled by the orchestration system or by an external logging solution. The recommended way to enable log rotation for a Prometheus container would depend on the specific orchestration system you are using and how you are running the container.
